package com.atharva.auth.user.utils.encrypt;

public class EncryptionException  extends RuntimeException  {

    public EncryptionException(String message, Throwable cause) {
        super(message, cause);
    }
}
